packages/astro/src/core/create-vite.ts

```diff
@@ -122,7 +122,11 @@ export async function createVite(
 				`${srcDirPattern}pages/**/_*.{js,mjs,ts,mts}`, // Remaining JS/TS files prefixed with `_` (not endpoints)
 				`${srcDirPattern}pages/**/_*/**/*.{js,mjs,ts,mts}`, // Remaining JS/TS files within directories prefixed with `_` (not endpoints)
 			],
-			exclude: ['astro', 'node-fetch'],
+			exclude: [
+				'astro:dev-toolbar',
```

Comment:
Is it potentially an issue that this only exclude parts of the dev toolbar?

Reply:
I'm not sure, what gets imported in the browser? is it a single entrypoint? Are there multiple? I tested this with the toolbar enabled and it works and does not work if I don't exclude this. Is there some scenarios I should try?

Reply:
Actually I think I can do this with a single `include` instead, testing now.

Reply:
The main entrypoint is at `astro/runtime/client/dev-toolbar/entrypoint.js`, that entrypoint imports `astro:dev-toolbar`

Reply:
Updated this so that `astro` is still excluded by default.